Dependencies/xr/Include/XR.h

```diff
@@ -140,6 +140,12 @@ namespace xr
                     Pose Pose;
                 };
 
+                struct JointSpace : Space
```

Comment:
nit: put the `bool` after the `float`. The compiler is likely padding out to 32 bits right now.